lib/apiservers/engine/backends/container.go

```diff
@@ -58,6 +58,19 @@ import (
 
 const (
 	bridgeIfaceName = "bridge"
+
```

Comment:
In Bville there was an alignment to 128MB - I think this was specifically for hotadd given the vmfork usage, but it's a reasonable constraint for us atm anyway given the bank size. If it's not functional with 512MB as a min then we really, really need to look into why, we should be able to go smaller.

Checks from bville:
```
	// validity checks
	if remainder := module.memoryMB % 128; remainder != 0 {
		log.Warnf("Default container VM memory must be 128 aligned for hotadd, rounding up.")
		module.memoryMB += 128 - remainder
	}

	// if we fail here it causes crashing in docker - it'd be nice to push this back to the installer
	// but we currently have no mechanism to do so
	if module.memoryMB < 384 {
		log.Warnf("Minimum memory for container template is 384MB, adjusting up.")
		module.memoryMB = 384
	}
```